### ISSUE TITLE
i2cdetect not working

### ISSUE THREAD

**Issue by seafrog1** [clearlinux/distribution#1531]
maybe the reason is that the /dev/i2c-x dirs are not in place
thx

**Comment by ahkok** [comment]
We ship all the standard `udev` rules. If you have any idea whether we need additional rules or something to enable this, please do tell. (Kernel config options? What devices specifically are you expecting to appear here?).

**Comment by fenrus75** [comment]
also this level of direct hardware access is not great and I would prefer
it to not be enabled.

**Comment by seafrog1** [comment]
I agree that direct hardware access can be dangerous. I use i2c programing in C on SBCs.
On the "big" machine i used i2cdetect before with fedora. So I installed it on CL via the bundle. 
Sorry, but in my opinion, if you provide a function it it should work. 
And you can not take all risks from users. Otherwise you should also not allow to sudo rm -rfd ...

Btw, a lot of thanks for providing CL

**Comment by fenrus75** [comment]
we need to figure this out

on the one hand, we provide the sensors stuff mostly for the other
functionality in the upstream, and this thing is a bit more dangerous than
that

on the other hand "real root" should get what it asks for, even if it's a
bit dangerous.... but ideally it's flagged as such somehow so that people
who don't understand the danger accidentally think it's safe when it's not.
